docs/qa/v034/README.md

```diff
@@ -560,7 +575,7 @@ Version: xxxxxxxxxxxxxxxxx
 
 #### 2/3 Tendermint Core - 1/3 CometBFT
 
-**Result: ????**
+**Result: PASS**
 
 Date: 2023-02-08
 
```

Comment:
Please change the "version" to "versions", and add
CometBFT: 3b783434f26b0e87994e6a77c5411927aad9ce3f (if that's the one you used)
Tendermint Core: [add here the hash of the Tendermint Core version]